src/server/api/cla.js

```diff
@@ -582,7 +579,8 @@ async function prepareForValidation(item, user) {
     for (let i = needRemove.length - 1; i >= 0; --i) {
         user.requests.splice(needRemove[i], 1)
     }
-    user.save()
+    await userService.save(user);
```

Comment:
@codezork do we need an `await` here? Can we just continue processing without waiting for user to be saved?

Reply:
I found it more save. Meanwhile I found no reason to wait. Removed now.